core/domain/skill_services.py

```diff
@@ -762,6 +762,24 @@ def get_skill_rights(skill_id, strict=True):
     return get_skill_rights_from_model(model)
 
 
+def get_multi_skill_rights(skill_ids):
+    """Retrieves the rights objects for the given skills.
+
+    Args:
+        skill_ids: list(str). Skill IDs of the skills for which rights are
+            requested.
+
+    Returns:
+        list(SkillRights). The list of skill rights objects.
+    """
+
+    skill_rights_models = skill_models.SkillRightsModel.get_multi(skill_ids)
+    skill_rights_list = [
```

Comment:
So there is a potential problem in this.
If suppose we ask for 3 skills for IDS (a,b,c)
and we return only skills for (a,c) because b doesn't exist in the backend. Right now will that throw an error? 
Because what you will return is [SkillRight(a), SkillRight(c)], instead you probably want [SkillRight(a), None, SkillRight(c)]

Reply:
I'm not sure but I think it will include None in the list? 
`Returns:
            list(*|None). A list that contains model instances that match
            the corresponding entity_ids in the input list. If an instance is
            not found, or it has been deleted and include_deleted is False,
            then the corresponding entry is None.`

https://github.com/oppia/oppia/blob/develop/core/storage/base_model/gae_models.py#L114

Reply:
Done.